packages/codemods/transforms/v3/dropdown/README.md

```diff
@@ -0,0 +1,26 @@
+# v3/dropdown
+
+
+## Usage
+
+```
+npx @hashicorp/design-system-codemods v3/dropdown path/of/files/ or/some**/*glob.hbs
+
+# or
+
+yarn global add @hashicorp/design-system-codemods
+@hashicorp/design-system-codemods v3/dropdown path/of/files/ or/some**/*glob.hbs
+```
+
+## Local Usage
+```
+node ./bin/cli.js v3/dropdown path/of/files/ or/some**/*glob.hbs
+```
```

Comment:
I would do something like this to make it clearer what options they have (I can't write the exact markdown, because of the multiple backticks, but I hope it gives you the gist of it):

## Usage

### Using `npx`
```
npx @hashicorp/design-system-codemods v3.0.0/dropdown path/to/single/file.hbs
```
or
```
npx @hashicorp/design-system-codemods v3.0.0/dropdown path/to/some/glob/**/*.hbs
```

### Global installation
```
yarn global add @hashicorp/design-system-codemods
```
and then
```
@hashicorp/design-system-codemods v3.0.0/dropdown path/to/single/file.hbs
@hashicorp/design-system-codemods v3.0.0/dropdown path/to/some/glob/**/*.hbs
```

### Local installation
```
node ./bin/cli.js v3.0.0/dropdown path/to/single/file.hbs
node ./bin/cli.js v3.0.0/dropdown path/to/some/glob/**/*.hbs
```

Reply:
(we may also want to limit the proposed approaches to only one or two)

Reply:
Or maybe here we mention only one of the methods (with the two file paths) and we link to the `packages/codemods/README.md` for details about other possible methods.

Reply:
Updated with a simplified version, suggesting npx for consumers and local binary for maintainers. Agree there were to many options